### PR TITLE
added files and acknowledgment

### PR DESCRIPTION
**Pull request by ElShaer398**

### CODE REVIEW

**Comment by ElShaer398** [comment]
Hi,
Very good efforts working on the project, good work, but there are still some things you need to work on before going to the next project.👌✌️

1. you should use linters for clean code.🤔
2. README file needs to describe your project.
3. Please take a look at the comments down below.👇

File: style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
+html, body {
+  height: 100vh;
```
Comment (ElShaer398) [inline style.css:2]: please don't use fixed heights because it's not good practice.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+#products{
+  display: flex;
+  flex-direction: row;
+  margin: 10px;
+  
+}
+
+#submit-button{
```
Comment (ElShaer398) [inline style.css:106]: Please use correct css to position your buttons.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+#header {
+
+  position: fixed;
+  top: 0%;
+  
+}
+
+#nav-bar{
+
+background-color:yellow;
```
Comment (ElShaer398) [inline style.css:17]: The color is not comfortable with eyes in the page, please stick to the template colors.

File: style.css (hunk 1 above)
Comment (didierganthier) [reply style.css:2]: Hey, nice one 😄 but try to communicate in a softer way, and give suggestions, not orders. Could also include some emojis to make it look more human.

File: style.css (hunk 3 above)
Comment (didierganthier) [reply style.css:17]: Same thing I suggested earlier can be applied here 😉

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @ElShaer398 ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Used inline comments ✔️ 

## Required Changes 🛠
- [ ] The best way to review is to use the `Review changes` feature, try using it for a better review
Screenshot 2023-02-25 at 21 52 54

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: style.css (hunk 1 above)
Comment (Shedrack-Sunday) [reply style.css:2]: - [Optional] Good job making the desired changes. however, incase of tags or signs such as %, use them with back strings to come out as `%`, to show their significance 👍

File: style.css (hunk 3 above)
Comment (Shedrack-Sunday) [reply style.css:17]: - [ ] Good job here 👏 . However, even though you included a "please" at the second part of the sentence, the first still looks like a command. also, always give a reason why the user should "stick to the template colors" in detail to guide the project owner.

Example of rephrasing that:

```Thank you for making efforts to your code 👏 . Based on my review, I have noticed that the color used on the page is not optimal for comfortable viewing. To ensure consistency with the template design and improve user experience, I recommend adhering to the template colors.```

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @ElShaer398      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


  
# Required Changes ♻️

- [x] I made detailed and well-explained suggestions on one of your reviews below based on the comments from the previous reviewer. This was made to have a professional review guide
  👍 


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

- [ ] Though missed by the previous reviewer, kindly always have a description of your project on the pull request. this makes your project have a professional look 

## View:
image

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @christianonoh 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


- [ ] As I said earlier, though missed by the previous reviewer, kindly always have a description of your project on the pull request. this makes your project a professional look 

## View:
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**